images/algod/DevModeNetwork.json

```diff
@@ -21,7 +21,8 @@
                 "Online": true
             }
         ],
-        "DevMode": true
+        "DevMode": true,
+        "RewardsPoolBalance": 0
     },
```

Comment:
@barnjamin Minor:  Unless opposed, can the README be extended to document the change?

I opened a child PR with a proposal:  https://github.com/algorand/sandbox/pull/150.

Reply:
good call! merged it in